lib/read_only/lfric/Makefile

```diff
@@ -49,22 +49,24 @@
 # Compiler and compiler flags
 F90 ?= gfortran
 F90FLAGS ?=
-# Path to the LFRic infrastructure library. It defaults to the pared-down
+# Path to the LFRic infrastructure library. It defaults to the included
 # version of the LFRic infrastructure distributed with PSyclone.
 # Overwrite for a different infrastructure version.
-LFRIC_INF_DIR ?= ./../../../src/psyclone/tests/test_files/lfric/infrastructure
+LFRIC_PATH ?= ./../../../external/lfric_infrastructure/src
 # Paths to the PSyclone wrapper libraries and required templates. They default
 # to the relative paths to the top-level 'lib' and 'lib/read_only'
 # directories. Overwrite for a custom location.
 PSYDATA_LIB_DIR ?= ./../..
 LIB_TMPLT_DIR ?= ./..
 # -----------------------------------------------------------------------------
 
-INF_INC = $(LFRIC_INF_DIR)
 INF_LIB_NAME = lfric
-INF_LIB = $(LFRIC_INF_DIR)/lib$(INF_LIB_NAME).a
+INF_LIB = $(LFRIC_PATH)/lib$(INF_LIB_NAME).a
+
+include $(LFRIC_PATH)/lfric_include_flags.mk
+
+F90FLAGS += $(LFRIC_INCLUDE_FLAGS) $$(nf-config --fflags)
```

Comment:
Why does the read-only library depend on netcdf? (Although, I suppose LFRic depends on netcdf so it's not a problem really.)

Reply:
Good point, it doesn't. Removed.